### ISSUE TITLE
storage-resize-images extension sometimes doesn't keep the original image orientation.

### ISSUE THREAD

**Issue by gki** [firebase/extensions#310]
###  Describe your configuration

- Extension name: `storage-resize-images`
- Extension version: 0.1.7
- Configuration values (redact info where appropriate):
```
Cloud Functions location: asia-northeast1
Cloud Storage bucket for images: omit
Sizes of resized images: 200x200
Deletion of original file: false
Cloud Storage path for resized images (Optional):
Cache-Control header for resized images (Optional):
```

#### Steps to reproduce:
1. Upload image via Firebase Strorage console.
2. Wait and reload console until the resized image file has been shown.
3. Check the resized image.

##### Expected result
The orientation of resized image is same with the original image.

##### Actual result
The orientation of resized image is not same.

**Original**
<img width="1016" alt="スクリーンショット 2020-05-17 16 25 31" src="https://user-images.githubusercontent.com/1529390/82138395-16315880-985b-11ea-81a2-269a9ad4a112.png">

**Resized**
<img width="1009" alt="スクリーンショット 2020-05-17 16 25 40" src="https://user-images.githubusercontent.com/1529390/82138399-19c4df80-985b-11ea-8a58-2a7eabf0ee12.png">

### Other Notes
- I tried to upload some images both `jpg` and `png` and `tumbler.jpg` doesn't has this issue.
- However, other `jpg` image (`mainImage.jpg`) has this issue.

### Resources
Here are images I tested.
##### images have this issue.
- `thumbler.png`
![tumbler](https://user-images.githubusercontent.com/1529390/82138467-a4a5da00-985b-11ea-96fe-4dcbe8d76c3c.png)

- `mainImage.jpg`
![mainImage](https://user-images.githubusercontent.com/1529390/82138474-af606f00-985b-11ea-8a8d-7b40f94126fe.jpg)

##### image doesn't have this issue.
- `thumbler.jpg`
![tumbler](https://user-images.githubusercontent.com/1529390/82138473-abcce800-985b-11ea-92bf-13b3b4196d18.jpg)

**Comment by russellwheatley** [comment]
Hi @gki,
               My guess is your tumbler image has EXIF data that the browser is using to orient your image properly. The resized image loses the EXIF data, and the browser renders the image incorrectly. This [article](https://www.howtogeek.com/254830/why-your-photos-dont-always-appear-correctly-rotated/) has a pretty good explanation as to why this is happening. 

This PR #306 will rotate the resized images to their proper orientation so that you will not have to rely on the medium using the image's EXIF data to render it properly.

**Comment by gki** [comment]
Hi @russellwheatley ,
Thanks for checking and sharing info.

OK, I can wait PR #306.
I hope your update will release sooner! 🤞

**Comment by laurenzlong** [comment]
This has now been released in v0.1.8 of storage-resize-images.
